createmobilespec/createmobilespec.js

```diff
@@ -365,7 +365,7 @@ shelljs.config.fatal = true;
 if (argv.clearnpmcache) {
     // clean out cached platforms and plugins and app-hello-world
     var home_dir = process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE;
-    shelljs.rm("-rf", path.join(home_dir, ".cordova/npm_cache"));
+    shelljs.rm("-rf", path.join(home_dir, ".cordova" + path.sep + "npm_cache"));
```

Comment:
you can just use provide ".cordova" and "npm_cache" as separate arguments to `path.join` here - `join` will handle using the proper separator.